plugins/vfake.py

```diff
@@ -6,11 +6,20 @@
 
 option('locale', 'en_US', 'default locale to use for Faker', replay=True)
 
+# See also: https://faker.readthedocs.io/en/master/communityproviders.html
+option('vfake_extra_providers', [], 'list of additional Provider classes to load via add_provider()', replay=True)
```

Comment:
It's not implemented yet, but I'm thinking that having a list as option default will mean "one of these enumerated values".  Maybe change the default to `None`? (which allows any type)

Reply:
Ahh good tip, thanks! I changed the default, and then moved some stuff around because it was starting to make `setValuesFromFaker` look noisy.